priv/templates/phx.gen.auth/auth.ex

```diff
@@ -121,28 +122,28 @@ defmodule <%= inspect auth_module %> do
   end
 
   <%= if live? do %>@doc """
-  Handles mounting and authenticating the current_<%= schema.singular %> in LiveViews.
+  Handles mounting and authenticating the current_scope in LiveViews.
 
   ## `on_mount` arguments
 
-    * `:mount_current_<%= schema.singular %>` - Assigns current_<%= schema.singular %>
+    * `:mount_current_scope` - Assigns current_scope
```

Comment:
Shouldn't this mirror `fetch_current_scope_for_...` and become `mount_current_scope_for_...`?

Reply:
we can change it, but since LV hooks don't have naming conflicts (you always need to pass the module), it's not really an issue

Reply:
Ah, ok, sounds good to me!